cluster/v1beta1/types_placement.go

```diff
@@ -292,3 +292,11 @@ type PlacementList struct {
 	// Items is a list of Placements.
 	Items []Placement `json:"items"`
 }
+
+const (
+	// PlacementDisableAnnotation is used to disable scheduling for a placement.
+	// The typical case is a placement is created in the global hub cluster.
```

Comment:
I would not mention global hub here, others will lose the context.

Reply:
Done

Reply:
could we just stay 

PlacementDisableAnnotation is used to disable scheduling for a placement. It is a experimental flag do let placement controller ignore this placement, so other placement consumers can chime in.